libs/ast_generic/AST_generic.ml

```diff
@@ -581,7 +581,7 @@ and qualified_info = {
 
 and qualifier =
   (* Java/C++/Rust *)
-  | QDots of (ident * type_arguments option) list
+  | QDots of (ident * type_arguments option) list * id_info
```

Comment:
No strong opinion, but I do wonder if it wouldn't be simpler and more general to just have `NameAttr` take an `expr`. We have another ticket/request for supporting sym-prop for attributes. The idents in the `QDots` are accompanied by `type_Arguments` which to me suggests that they are not meant to encode expressions.

Reply:
Yes maybe the issue here is how we currently encode app.get as a qualified name when really it's a DotAccess
One issue is that in Python it's not super clear when you see foo.bar.z
whether bar is a module, or a field.
So probably we should go more general and use DotAccess here.

Reply:
maybe we can have an ExprAttr of ...
or encode it via OtherAttribute.